add_db_backup.js

```diff
@@ -0,0 +1,132 @@
+const fs = require('fs');
+const rimraf = require('rimraf');
+const exec = require('child_process').exec;
+
+const d = new Date();
+const dN = new Date(2018, 6, 24);
+const todaysDate = `${d.getFullYear()}-${d.getMonth() + 1}-${d.getDate()}-${d.getHours()}:${d.getMinutes()}:${d.getSeconds()}`;
+// const testDate = `${d.getFullYear()}-${d.getMonth() + 1}-${d.getDate() -8}`;
+// const pastDate = `${d.getFullYear()}-${d.getMonth() + 1}-${d.getDate() - 7}`;
+
+
+if (fs.existsSync('./db_backup')) {
```

Comment:
For precautious always use deterministic paths. If NPM called outside root dir a 'db_backup' dir would be created in current_dri/db_backup. Try to use ```__dirname + '/db_backup'```